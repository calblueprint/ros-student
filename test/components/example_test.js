import React from 'react';
import { Link } from 'react-router';
import renderer from 'react-test-renderer';

it('changes the class when hovered', () => {
  const component = renderer.create(
    <Link page="http://www.facebook.com">Facebook</Link>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
