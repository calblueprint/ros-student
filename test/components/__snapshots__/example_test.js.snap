exports[`test changes the class when hovered 1`] = `
<a
  onClick={[Function bound handleClick]}
  page="http://www.facebook.com"
  style={Object {}}>
  Facebook
</a>
`;
